packages/coinstac-common/src/services/classes/db-registry.js

```diff
@@ -189,11 +188,23 @@ class DBRegistry {
 
     conf.pouchConfig.ajax = result(this, 'ajax');
 
-    // convert string declared PouchDB adapter to adapter instance
-    const requestedAdapter = get(conf, 'pouchConfig.db');
-    /* istanbul ignore if */
-    if (isString(requestedAdapter)) {
-      set(conf, 'pouchConfig.db', require(requestedAdapter)); // eslint-disable-line
+    /**
```

Comment:
This is _pretty gross_. We can hopefully clean up when we split `DBRegistry` into its own package.